src/components/common/view/Avatar.js

```diff
@@ -34,17 +34,22 @@ const CustomAvatar = ({ styles, style, source, onPress, size, imageSize, childre
       activeOpacity={1}
       disabled={!onPress}
       onPress={_onPress}
-      style={[styles.avatarContainer, { width: size, height: size, borderRadius: size / 2 }, style]}
+      style={[styles.avatarContainer, { width: size, height: size, borderRadius: '50%' }, style]}
```

Comment:
border radius isnt supported on react native

Reply:
@sirpy 

1. because dashboard and privacy screens aren't used Avatar component. i refactored to use it for have single "source of trust" to load/display avatars. Why component not the hook - because it encapsulates everything, with the hook you would need to import undefined profile svg, good dollar logo etc. Also using of the universal component follows the React's DRY principle
2. they used an Image component with avatar base64 passed as `{uri:}` source
3. i restored all the styles. Now the pages are looking the following way:

<img width="467" alt="GoodDollar | PROFILE PRIVACY 2021-06-09 13-26-40" src="https://user-images.githubusercontent.com/62208067/121338743-78597180-c926-11eb-8225-b797eea89707.png">
<img width="468" alt="GoodDollar | Wallet 2021-06-09 13-25-43" src="https://user-images.githubusercontent.com/62208067/121338756-7a233500-c926-11eb-930d-a8ef6cf2b1f8.png">

This need to be fixed.